src/main/java/nextstep/step2/vo/Name.java

```diff
@@ -0,0 +1,51 @@
+package nextstep.step2.vo;
+
+import java.util.Objects;
+
+public class Name {
+    private static final String NOT_NULL_OR_EMPTY_EXCEPTION_MESSAGE = "이름은 빈 값이거나 null 일 수 없습니다.";
+    private static final String RANGE_EXCEPTION_MESSAGE = "이름은 1~5자 입니다.";
+
+    private final String value;
+
+    private Name(String value) {
+        this.value = value;
+    }
+
+    public static Name create(String value) {
+        if(value == null || value.isEmpty()) {
+            throw new IllegalArgumentException(NOT_NULL_OR_EMPTY_EXCEPTION_MESSAGE);
+        }
+
+        if(value.length() < 1 || value.length() > 5) {
+            throw new IllegalArgumentException(RANGE_EXCEPTION_MESSAGE);
+        }
```

Comment:
null 체크와 빈값에 대한 체크를 해주셔서 다음과 같이 해주셔도 좋을 것 같아요! 
또한 매직넘버를 사용하셨는데 상수화를 통해 값에 의미를 부여해주면 어떨까요? 😄 
```suggestion
        if(value.length() > 5) {
            throw new IllegalArgumentException(RANGE_EXCEPTION_MESSAGE);
        }
```